docs/modules/ROOT/pages/volume.adoc

```diff
@@ -0,0 +1,60 @@
+= Volume
+
+TODO: Expand severely.
+
+lb-operator acts as a CSI `PersistentVolume`, which helps it stabilize network addresses, inject pod metadata, and exposing individual pods.
+
+== Stable addresses
+
+Some xref:loadbalancerclass.adoc[] strategies, such as `NodePort`, tie the public address to the Kubernetes node that the `Pod` is running on. When this address must be configured statically in clients
+(such as for HDFS NameNodes), then Kubernetes' default "floating" scheduling either requires all clients to be reconfigured every time something moves, or for all clients to proxy their traffic through
+a single static node, which then becomes a single point of failure (along with the node that the workload is running on).
+
+Mounting load balancers into Pods as `PersistentVolume` allows lb-operator to pin these workloads to one node. Note that this only happens for xref:loadbalancerclass.adoc[] es that actually benefit
+from pinning.
+
+== Pod metadata injection
+
+Some services (such as Kafka) need to know their external address, so that they can advertise it to their own replica discovery mechanism. xref:loadbalancer.adoc[] volumes contain a file tree that exposes
+this information:
+
+[source]
+----
+- `default-address/`- symlink to `addresses/\{primary address\}`
+- `addresses/` - Contains information about all addresses associated with this xref:loadbalancer.adoc[]
+  - `\{address\}/` - A folder is created for each address
+    - `address` - Contains the Pod's address (IP address or hostname)
+    - `ports/` - Contains port numbers for each named port
+      - `\{port name\}` - Contains the public port number for this named port
+----
+
+== Individual pod exposure
+
+Sometimes each replica much be exposed individually, for example because clients need to access data on a specific shard. `PersistentVolumeClaim` templates can be used to provision this automatically.
```

Comment:
```suggestion
Sometimes each replica must be exposed individually, for example because clients need to access data on a specific shard. `PersistentVolumeClaim` templates can be used to provision this automatically.
```